app/static/lib/conp-react/src/DatasetElement/index.js

```diff
@@ -48,250 +47,259 @@ const DatasetElement = (props) => {
+                   className="btn btn-success m-1">
+                Download With DataLad
+              </a>
+            </div>
+            <div className="col-2 p-2">
+              <p className="card-text pl-1">
+                <FontAwesomeIcon
+                  icon={faQuestionCircle}
+                  color="dimgray"
+                  size="lg"
+                  onMouseEnter={handleDataLadMouseEnter}
+                  onMouseLeave={handleDataLadMouseLeave}
+                  data-tip
+                  data-for="dataladTip"
+                />
+                {showDataLadTipText &&
+                  <ReactToolTip id="dataladTip" multiline={true} style={{ Width: "70px", WhiteSpace: "pre-wrap" }}>
+                    DataLad ...
+                  </ReactToolTip>
+                }
+              </p>
+            </div>
+          </div>
+
+          <div className="row w-100 align-items-center">
+            <div className="col-10 p-0">
+              {element.cbrain_id ?
+                <button onClick={openCbrainModal} className="btn btn-outline-success m-1">
+                  <div class="d-flex row align-items-center justify-content-center">
+                    Process On <img
+                        className="cbrain-img justify-content-center align-items-center pl-4"
+                        src="static/img/cbrain-long-logo-blue.png" style={{maxHeight: '30px'}}
+                    />
                   </div>
-                  <a href={`dataset?id=${element.id}#downloadInstructions`} role="button"
-                     className="btn btn-outline-secondary m-1">
-                    Dataset
-                    <div className="spinner-border text-primary" role="status" hidden={!datasetSpinnerState}>
-                      <span className="sr-only">Loading...</span>
-                    </div>
-                  </a>
-                  <div className="alert alert-danger" role="alert" hidden={!datasetErrorState}>
-                    {datasetErrorText}
+                </button> :
+                <button className="btn btn-outline-secondary disabled m-1">
+                  <div className="d-flex row align-items-center justify-content-center">
+                    Process On
+                    <img
+                        className="cbrain-img justify-content-center align-items-center pl-4"
+                        src="static/img/cbrain-long-logo-grey.png" style={{maxHeight: '30px'}}/>
                   </div>
-                </div>
-                <div className="d-flex">
-                  {authIcons.map((icon, index) => <div key={"authIcon_" + index}
-                                                       className="text-center p-1">{icon}</div>)}
-                </div>
-              </div>
+                </button>
+              }
+            </div>
+            <div className="col-2 p-2">
+              <p className="card-text pl-1">
```

Comment:
Maybe make this a `div`? React complains for me in the dev console about having a `ReactToolTip` as a descendant of a `p`.